clusterloader2/pkg/prometheus/prometheus.go

```diff
@@ -57,6 +57,7 @@ func InitFlags(p *config.PrometheusConfig) {
 	flags.BoolEnvVar(&p.ScrapeNodeExporter, "prometheus-scrape-node-exporter", "PROMETHEUS_SCRAPE_NODE_EXPORTER", false, "Whether to scrape node exporter metrics.")
 	flags.BoolEnvVar(&p.ScrapeKubelets, "prometheus-scrape-kubelets", "PROMETHEUS_SCRAPE_KUBELETS", false, "Whether to scrape kubelets. Experimental, may not work in larger clusters. Requires heapster node to be at least n1-standard-4, which needs to be provided manually.")
 	flags.BoolEnvVar(&p.ScrapeKubeProxy, "prometheus-scrape-kube-proxy", "PROMETHEUS_SCRAPE_KUBE_PROXY", true, "Whether to scrape kube proxy.")
+	flags.StringEnvVar(&p.SnapshotProject, "experimental-snapshot-project", "PROJECT", "", "GCP project used where disks and snapshots are located.")
```

Comment:
I think you need to default it somewhere to not break current behavior.

Reply:
@wojtek-t we rely on the `$PROJECT` environment variable being set - this is done by kubetest [here](https://github.com/kubernetes/test-infra/blob/5f62df0ca89fd93fdcc3f1a7a76019bb115ffa47/kubetest/main.go#L775).

Reply:
It's either the value of `--gcp-project` flag being passed to kubetest or the project being assigned by boskos (if `--gcp-project` was not provided).

Reply:
Resolving after offline discussion.